project/settings.py

```diff
@@ -14,11 +14,10 @@
 
 from django.contrib.messages import constants as messages
 
-MODE = "prod"  # Overridden by local settings.
 ENVIRONMENT = os.environ.get("ENVIRONMENT", "develop")
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Try
```python
Path(".").parent.resolve()
```

Reply:
switched to using Path, see above about why not using `.`

Reply:
Actually though - upon going through changing the paths to use Path, I'd like to push back on the need to switch right now:

* These settings are expected in a variety of places (including Django code) to be strings rather than Path objects. So we need to either be careful to cast them to strings whenever passing them (e.g., call to `static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)`) or need to store them as strings. The latter is doable enough - use Path to construct the path, cast to a string to store, then construct new paths like `str(Path(BASE_DIR, 'subdirectory'))`. But Path objects by design always strip trailing slashes, so we'd be back to using os.path anyway to construct e.g. `MEDIA_ROOT`. 

* Upon considering the various settings for local assets (e.g. `STATICFILES_LOCATION = "/static"`), we might also need to update those to be filesystem-independent - but that would involve changing usage in templates too...

Overall this just seems like a separate endeavor and, although potentially worth doing, not a priority right now - is that ok?

Reply:
hmmmm... In theory, since we use Python 3.6+, Path objects should be acceptable wherever strings used to be (for example, `os.path.join` accepts `Path` objects as well)...

... Ah, I see - we need to upgrade to 3.1 if we want [Path objects to work for stuff like staticfiles dirs](https://github.com/django/django/commit/77aa74cb70dd85497dbade6bc0f394aa41e88c94). This is a bug that's been fixed.

Up to you - if you decide to defer the `Path` conversion can you make a ticket for it and assign it to me?

Reply:
Made a ticket for it!